### ISSUE TITLE
Do not delete session records from the database

### ISSUE THREAD

**Issue by juandaherrera** [masenf/reflex-local-auth#10]
I was thinking it would be a good idea not to delete these records and instead use a boolean to indicate whether the session is still active. Additionally, we could add a field to indicate when the session started. What do you think of this idea? Do you think it's feasible?

This would allow us to persist and have traceability for all login sessions.

**Comment by masenf** [comment]
Yeah, that is the approach that will be introduced with "reflex-auth-base" which will handle sessions, groups, and permissions so that auth plugins like reflex-local-auth can just focus on authentication.

It's coming.

**Comment by juandaherrera** [comment]
That sounds pretty good. Is there any repo where you guys are working? Just to see and understand if that fits what I'm working on

**Comment by albertsola** [comment]
I'm planning to do some work on reflex/auth that I could opensource or add as contribution to reflex-local-auth.

Regarding the session, I was considering moving to a JWT token. The reasons behind it, is to allow non users to be granted access to protected areas/records with a specifically crafted token.

For example: I could generate one JWT token that allows you to change the password of a specific user, this JWT token expires in x hours, and can be send by email with a link to allow a user to reset their password.

@masenf it would be a good idea to put together a small roadmap covering `reflex-local-auth` and `reflex-auth-base` so we can start contributing.

Until we have the roadmap, I will create some feature requests where we can have a discussion on how to best fulfill them.
